python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -36,6 +36,70 @@
 
 import Gaffer
 import GafferScene
+import GafferUI
+
+##########################################################################
+# UI for the state plug that allows setting the state through buttons
+##########################################################################
+
+from collections import namedtuple
+
+ButtonLabels = namedtuple( 'ButtonLabels', 'startButton pauseButton')
+
+class InteractiveRenderingStateWidget( GafferUI.PlugValueWidget ) :
+
+	labelMap = {
+		GafferScene.InteractiveRender.State.Stopped : ButtonLabels('Start', 'Pause'),
+		GafferScene.InteractiveRender.State.Running : ButtonLabels('Stop', 'Pause'),
+		GafferScene.InteractiveRender.State.Paused : ButtonLabels('Continue', 'Paused') }
+
+	def __init__( self, plug, *args, **kwargs) :
+
+		self.plug = plug
+
+		row = GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Horizontal )
+		GafferUI.PlugValueWidget.__init__( self, row, plug )
+
+		with row :
+			self.startButton = GafferUI.Button()
```

Comment:
Prefix with `__`, to make it clear that this is an internal implementation detail. Same reason as above.